torch/csrc/autograd/python_variable.cpp

```diff
@@ -154,6 +154,13 @@ static bool check_has_torch_dispatch(PyObject *obj) {
   );
 }
 
+static PyObject* getXLATensorClass() {
+  // TODO: What should we do if XLATensor doesn't exist? Return THPVariableClass?
+  // TODO: is THPObjectPtr the right pointer container?
+  static THPObjectPtr xla_tensor_ptr(py::globals()["XLATensor"].ptr());
```

Comment:
I'm not familiar with the pybind globals API but how is the error raised when the key is not there? Should you handle it like a C-python error? Or is it a c++ error?

Reply:
@albanD  We could check if the key exists.
```
static PyObject* xla_tensor_python_class = py::globals().contains("XLATensor) ? py::globals()["XLATensor"].ptr() : nullptr;
return xla_tensor_python_class ;
```

**I wonder what should we do if the key doesn't exist?** Would it be acceptable to fall back to THPVariableClass? cc: @ezyang or should we throw?

Reply:
When would that happen? If the users loads an older version of torch_xla that doesn't have that class?

Reply:
> If the users loads an older version of torch_xla that doesn't have that class?

Yeah, I'm not sure this is very likely, so maybe we shouldn't worry about being a backward compatible with older XLA versions and just throw? 

> When would that happen? 

A great q. I don't know really 😸  the XLA extension was partially initialized, so we somehow can create XLA cpp tensors, but some python classes are missing.